### PR TITLE
fix(pie-monorepo): DSW-1969 updated Playwright and Percy dependencies OLD

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
## Describe your changes (can list changeset entries if preferable)
- Updated Percy and Playwright dependencies
- Updated tests to support updated Playwright deps

A couple things to note with this PR is that all screenshots added are now covering the Chrome browser.
Some differences in the screenshots of existing packages are due to the change of order in some of the listed variations.

Another note is there is 1 test that has been skipped. This seems to be a legitimate failure of the component test that is blocking this work so a ticket will be written to address this.

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] I have reviewed visual test updates properly before approving

### Reviewer 1 - @xander-marjoram 
- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,12 +78,12 @@
     "@justeattakeaway/pie-webc-testing": "0.13.0",
     "@justeattakeaway/pie-wrapper-react": "0.14.1",
     "@justeattakeaway/stylelint-config-pie": "0.7.0",
-    "@percy/cli": "1.26.3",
-    "@percy/playwright": "1.0.4",
-    "@playwright/experimental-ct-react": "1.32.3",
-    "@playwright/test": "1.32.3",
+    "@percy/cli": "1.28.8",
+    "@percy/playwright": "1.0.5",
```
Comment (xander-marjoram) [inline package.json:82]: QQ (don't want to scope creep): Will we be updating to `1.0.6` soon?

File: packages/components/pie-button/test/visual/pie-button-size.spec.ts [hunk 2]
```diff
@@ -28,8 +29,10 @@ const renderTestPieButton = (propVals: WebComponentPropValues) => `<pie-button s
 const componentPropsMatrix : WebComponentPropValues[] = getAllPropCombinations(props);
 
 // eslint-disable-next-line no-empty-pattern
-test.beforeEach(async ({ }, testInfo) => {
+test.beforeEach(async ({ mount }, testInfo) => {
```
Comment (xander-marjoram) [inline packages/components/pie-button/test/visual/pie-button-size.spec.ts:32]: I think the eslint comment above this line can be removed now

File: packages/components/pie-modal/test/component/pie-modal.spec.ts [hunk 3]
```diff
@@ -417,14 +419,17 @@ test.describe('modal', () => {
                 expect(isModalVisible).toBe(true);
             });
 
-            test('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
+            test.skip('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
```
Comment (xander-marjoram) [inline packages/components/pie-modal/test/component/pie-modal.spec.ts:422]: Can you add a link to the ticket for this please?

Comment (JoshuaNg2332) [reply packages/components/pie-modal/test/component/pie-modal.spec.ts:422]: Can't link to Jira tickets in code but will drop the ticket number as a comment above.

File: package.json (hunk 1 above)
Comment (JoshuaNg2332) [reply package.json:82]: Updated and ran tests and looks all ok so will keep updated.

File: packages/components/pie-modal/test/component/pie-modal.spec.ts (hunk 3 above)
Comment (siggerzz) [reply packages/components/pie-modal/test/component/pie-modal.spec.ts:423]: Given we link to JIRA tickets in PR's, I think referencing the ticket number is okay here.

Comment (JoshuaNg2332) [reply packages/components/pie-modal/test/component/pie-modal.spec.ts:423]: Comment added. 👍🏼
